phases/19-capstone-projects/83-prompt-injection-detector/code/tests.py

```
@@ -0,0 +1,98 @@
+
+from __future__ import annotations
+
+import unittest
+
+from benign import prompts as load_benign
+from main import Detector, PerCategoryMetrics, Verdict, evaluate, load_taxonomy, normalize
+
+
+class TestNormalize(unittest.TestCase):
+    def test_zero_width_stripped(self) -> None:
+        text = "How do I R​E‌D‍ACTED?"
```

Ruff (0.15.14)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)